controllers/ControllerDecorator.ts

```diff
@@ -17,11 +17,19 @@ import {ErrorHandlerManager, ERRORHANDLER_KEY, DEFAULT_ERROR_HANDLER} from '../e
 import httpStatus = require('http-status');
 
 let controllers: ControllerRegistration[] = [],
-    definedRoutes = [];
+    definedRoutes = [],
+    bodyParserInstalled = false;
 
 const PRIMITIVE_TYPES = [Object, String, Array, Number, Boolean],
     NON_JSON_TYPES = [String, Number, Boolean];
 
+try {
+    require('body-parser');
```

Comment:
It does not ensure that the body-parser is registered in express... But I not sure if this is even possible...